apps/web/src/app/globals.css

```
@@ -5,6 +5,9 @@
 /* Import UI package global styles */
 @import "@workspace/ui/globals.css";
 
+/* OverlayScrollbars base styles (used by ScrollArea / OverlayScroll / root body) */
+@import "overlayscrollbars/overlayscrollbars.css";
```

Stylelint (17.9.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)